girder_worker/plugins/docker/tests/docker_test.py

```diff
@@ -42,6 +42,8 @@ def _mockOsRead(fd, *args, **kwargs):
 
 
 def setUpModule():
+    girder_worker.load_plugins()
+
```

Comment:
If `load_plugins` is lazily-loaded correctly (see prior comment), `load_plugins` should not be needed on any of the tests.